src/main/java/com/epam/model/Room.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Room {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private Integer number;
+
+    @NotNull
+    private String numberPlace;
+
+    @NotNull
+    private String costNight;
+
+    @ManyToOne
+    @JoinColumn(name = "room_type_id")
+    private RoomType roomType;
+
+    @JsonIgnore
```

Comment:
Since we're using DTOs we don't need JSON-related annotations in model.

Reply:
When we don't use this annotation we hava such error 
![default](https://user-images.githubusercontent.com/26812325/35208971-fe83f30e-ff5b-11e7-8f13-3f2f8d2e8ea6.PNG)

Reply:
Done